src/spec/author_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../Classes/author'
+
+describe Authors do
+  context 'would test Author class file' do
+    it 'should test the author class file' do
+      @author = Author.new('Assumpta', 'Alena', 'Okolike')
+      expect(@author.name).to eq('Assumpta')
+      expect(@author.first_name).to eq('Alena')
+      expect(@author.last_name).to eq('Okolike')
+    end
```

Comment:
- [ ] Kindly make sure that all tests run and pass successfully. Currently, there's an error that appears each time I try to run the tests. Please fix it and make sure that all the tests pass successfully.
![image](https://user-images.githubusercontent.com/62290545/202570762-19a5fde6-8d24-4073-b8e4-b5566a02e308.png)